app/views/notes/_note.gpx.builder

```diff
@@ -1,9 +1,14 @@
 xml.wpt("lon" => note.lon, "lat" => note.lat) do
+  xml.time note.created_at.to_s(:iso8601)
+  xml.name "#{t'note.mine.description'} #{note.id}"
```

Comment:
I suspect the `#{t'browse.note.title', :id => note.id}` which gives "Note: 42" would work much better?